python_modules/dagster-ext/json_schema/message.json

```diff
@@ -1,5 +1,9 @@
 {
   "properties": {
+    "dagster_ext_version": {
```

Comment:
I think this should be `__dagster_ext_version` to indicate this is not a "user-space" field in the protocol

Reply:
Agree, unfortunately it's a little more complicated than it should be because Pydantic treats `_`-prefixed properties differently when generating json schema, I'll figure it out.